source/kit/components/ui/msm-form-submit-button/msm-form-submit-button.html

```diff
@@ -1,3 +1,4 @@
 <button type="submit" class="btn btn-primary" ng-disabled="formCtrl.loading || formCtrl.$invalid">
-  <i class="zmdi zmdi-hc-fw {{ formCtrl.loading ? 'zmdi-hc-spin zmdi-spinner' : (icon || 'zmdi-check') }}"></i>&nbsp;{{::label | translate}}
+  <i class="zmdi zmdi-hc-fw {{ formCtrl.loading ? 'zmdi-hc-spin zmdi-spinner' : (icon || 'zmdi-check') }}"
+	 ng-class="{'hidden-xs': hideIconOnMobile}"></i>&nbsp;{{::label | translate}}
```

Comment:
The way it is implemented now, when the icon is hidden on mobile, the label will still contain the initial whitespace. Maybe add an extra span containing both the icon and the whitespace and add "hidden-xs" there. Not sure if this breaks any styles though...